doc/releases/quincy.rst

```diff
@@ -104,7 +633,6 @@ Changelog
 * ceph.spec.in: openSUSE: require gcc11-c++, disable parquet (`pr#46155 <https://github.com/ceph/ceph/pull/46155>`_, Tim Serong)
 * ceph.spec: fixing cephadm build deps (`pr#47069 <https://github.com/ceph/ceph/pull/47069>`_, Redouane Kachach)
 * cephadm/ceph-volume: fix rm-cluster --zap (`pr#47626 <https://github.com/ceph/ceph/pull/47626>`_, Guillaume Abrioux)
-* cephadm/mgr: adding logic to handle --no-overwrite for tuned profiles (`pr#47944 <https://github.com/ceph/ceph/pull/47944>`_, Redouane Kachach)
```

Comment:
This is 17.2.4's changelog. Is this removal intentional? Requesting-changes just in case, to prevent accidental merge.

Reply:
@ljflores I don't know about that ^ line, do you?